src/Sylius/Component/Translation/Model/AbstractTranslatable.php

```diff
@@ -19,15 +19,11 @@
 abstract class AbstractTranslatable implements TranslatableInterface
 {
     /**
-     * Translations.
-     *
      * @var TranslationInterface[]
```

Comment:
@michalmarcinkowski Sure, that's why I want to prevent coupling at the moment :)